packages/broker/test/integration/plugins/operator/deployOperatorContract.ts

```diff
@@ -52,3 +56,18 @@ export async function deployOperatorContract(
     const newOperator = new Contract(newOperatorAddress, operatorABI, deployer) as unknown as Operator
     return newOperator
 }
+
+export async function createWalletAndDeployOperator(provider: Provider, config: Chain, theGraphUrl: string, 
+    adminKey?: string): 
+    Promise<{ operatorWallet: Wallet, operatorContract: Operator, operatorConfig: OperatorServiceConfig }> {
```

Comment:
This should be moved to its own file `createWalletAndDeployOperator.ts`, or alternatively, this file re-named to suggest that it does not contain just one helper function.

Also multi-line params a bit clearer:
```suggestion
export async function createWalletAndDeployOperator(
    provider: Provider,
    config: Chain,
    theGraphUrl: string, 
    adminKey?: string
): Promise<{ operatorWallet: Wallet, operatorContract: Operator, operatorConfig: OperatorServiceConfig }> {
```